### ISSUE TITLE
Adjust position of latest year

### ISSUE THREAD

**Issue by sigmazen** [DanielYKPan/date-time-picker#459]
Hi
I am setting the maxDate to a date in the past (eg 18 years before today) which is working fine; as you can see from the first image the future dates are now disabled.
However, I would also like to move this new maxDate to the very bottom-right of the multiYears picker to enable more options when the User opens the picker.
Any thoughts?
Thanks in advance

ie from this:
![image](https://user-images.githubusercontent.com/7496847/44596639-cc006d80-a781-11e8-874e-07c475494aa5.png)

to this:
![image](https://user-images.githubusercontent.com/7496847/44596662-e8040f00-a781-11e8-8eb2-bba6173667d6.png)

**Comment by DanielYKPan** [comment]
Please use StackBlitz to build an app and show the issue you mention about
I would close this until it is confirmed as an issue.

**Comment by sigmazen** [comment]
Hi
Not so much an issue, but a question which I think stems from the yearView starting from `0` in grid and whether an offset could be implemented move this from the very first position of the grid to, say, the 12th position (blanks beforehand) to allow what I was asking.
The Use Case is for 16 years+ so anything newer than `today - 16 years` is not valid.

I'm using your StackBlitz demo, start-view folder.
I add `[max]="endDate"` into the `input` tag of start-view.component.html

If I set `endDate = new Date(1994, 0, 1);` in start-view.component.ts we can see that it appears the very last option in the dropdown, and the right arrow is disabled :-)
If I now set `endDate = new Date(1995, 0, 1);` we can see that it appears in the top-left and I can only go back in time so immediately need to left-arrow-click.

What I'd like to do is for this 1995 value to be in the bottom-right corner so all prior, eligible, years are shown.

I think this might be doable with an offset calculation to 'pad' year0 with XX leading empty values in grid (in my 1994 example this would be 0 empty cells, but in the 1995 example this would be 20 empty cells)
Thoughts?
Thanks again ... awesome module.
Cheers
